spec/support/shared_examples/dead_jobs.rb

```diff
@@ -26,39 +28,16 @@
     ]
   end
 
-  let(:dqt_response) do
-    { "trn" => "1234567",
-      "firstName" => "Bobby",
-      "lastName" => "Burger",
-      "middleName" => "Chrystal",
-      "dateOfBirth" => "1989-11-01",
-      "hasActiveSanctions" => false,
-      "initialTeacherTraining" =>
-      [{ "programmeStartDate" => "2022-09-03",
-         "programmeEndDate" => "2023-07-01",
-         "programmeType" => "InternationalQualifiedTeacherStatus",
-         "result" => "InTraining",
-         "provider" => { "ukprn" => "10007159" },
-         "husId" => nil,
-         "active" => true }] }
-  end
-
   let(:dqt_response_humanised) do
     <<~TEXT
-      trn: 1234567
-      firstName: Bobby
-      lastName: Burger
-      middleName: Chrystal
-      dateOfBirth: 1989-11-01
-      hasActiveSanctions: false
```

Comment:
Checked with Rob who mentioned we didn't have to sync the `middleName` and `hasActiveSactions` so have removed these.